index.html

```diff
@@ -19,8 +19,8 @@
       <h1 class="hero-heading">Cat Match</h1>
       <p class="hero-sub-heading">Find your perfect cat breed</p>
       <div class="hero-svg-section">
-        <img src="./assets/images/man_cat_couch.svg" class="hero-svg">
-        <button class="hero-button">Get started</button>
+        <img src="./assets/images/man_cat_couch.svg" class="hero-svg" alt="man and cat on couch">
```

Comment:
I haven't seen any sizings added for that image and on desktop it seems way too large, would you mind adding something a bit more appropriate for it?